Model/Ui/ConfigProvider.php

```diff
@@ -97,7 +97,7 @@ public function getConfig()
                         $plan['key'] = $planConfig->planKey();
                         $plan['logo'] = $planConfig->logoFileName();
                         $plan['installments'] = $pe->getEligibility()->getPaymentPlan();
-
+                        $plan['eligibility'] = $pe->getEligibility();
```

Comment:
maybe add a `@TODO` here about manipulating the API object in DTO instead of just take the necessary property

Reply:
add empty line before `return`